### PR TITLE
[KOGITO-9974] cluster-wide supporting services

### PR DESCRIPTION
**Pull request by tchughesiv**
https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/345

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `Issue-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] Issue-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by domhanak — APPROVED** [review]
Thank you

**Comment by domhanak** [comment]
@wmedvede @ricardozanini please check

**Review by wmedvede — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/supporting-services.adoc [hunk 1]
```diff
@@ -98,9 +98,32 @@ spec:
...
+kind: SonataFlowClusterPlatform
+metadata:
+  name: cluster-platform
+spec:
+  platformRef:
+    name: sonataflow-platform
+    namespace: <central namespace>
+----
+
+NOTE: These cluster-wide services can be overridden in any namespace, by configuring that namespace's `SonataFlowPlatform.spec.services`.
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/supporting-services.adoc:121]:
```suggestion
[NOTE]
====
These cluster-wide services can be overridden in any namespace, by configuring that namespace's `SonataFlowPlatform.spec.services`.
====
```

**Review by ricardozanini — APPROVED** [review]
Thank you!

**Pull request merged**